### PR TITLE
setup the server for employees

### PR DESCRIPTION
**Pull request by donovankg**
setup the end points, auth, added seed data, cru functions

### CODE REVIEW

File: server/api/employee/thing.events.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
```
Comment (cleverjam) [inline server/api/employee/thing.events.js]: This whole file is not needed

File: server/api/employee/thing.integration.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+'use strict';
```
Comment (cleverjam) [inline server/api/employee/thing.integration.js]: BAD!

These tests are for "api/things" not "api/employee"

Get rid of them, we don't need tests! not part of the story! 

/endrant

File: server/routes.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default function(app) {
   // Insert routes below
   app.use('/api/things', require('./api/thing'));
   app.use('/api/users', require('./api/user'));
-
+app.use('/api/employee', require('./api/employee'))
```
Comment (lchapoy) [inline server/routes.js]: Have proper indentation

File: server/api/employee/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+// not needed yet
```
Comment (lchapoy) [inline server/api/employee/index.spec.js]: You can erase the content of this file, you still have as example the real one

File: server/api/employee/employee.controller.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+}
+//
+// Creates a new Thing in the DB
+export function create(req, res) {
+  return Employee.create(req.body)
+    .then(respondWithResult(res, 201))
+    .catch(handleError(res));
+}
+
+// // Updates an existing Thing in the DB
```
Comment (lchapoy) [inline server/api/employee/employee.controller.js]: Same as above, comment to be employee representative

**Pull request merged**